src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1719,7 +1719,7 @@ export class HtmlVideoPlayer {
                     .isViewModeSupported(Windows.UI.ViewManagement.ApplicationViewMode.compactOverlay))
             // Check standard PiP support
             || document.pictureInPictureEnabled
-        ) {
+        ) && !browser.xboxOne) {
```

Comment:
I’m guessing this is currently falling under the check for windows? Maybe this church could be moved there?